.tekton/trillian-database-1-5-push.yaml

```diff
@@ -21,6 +20,8 @@ spec:
     value: Dockerfile.database
   - name: git-url
     value: '{{repo_url}}'
+  - name: image-expires-after
+    value: 5d
```

Comment:
This should only exist in the pull request pipelines, not the push pipelines. When code is pushed to the production branch, a build is triggered on RHTAP, and any build that was generated from a push event is a release candidate and shouldn't expire.
```suggestion
```

Reply:
ack, changing